dtos/common/ping.go

```diff
@@ -25,3 +26,11 @@ func NewPingResponse() PingResponse {
 		Timestamp:   time.Now().Format(time.UnixDate),
 	}
 }
+
```

Comment:
This code is only used by our services and SDKs. I think we can just add the service name parameter to the existing NewPingResponse(). i.e. it is not a breaking change since the code that uses it is internal. @cloudxxx8 do you agree?

Reply:
It is fine with me, but don't you think we should use Version API instead of Ping?

Reply:
@cloudxxx8 , good point. We could add it to both or all the common DTOs (Ping, Version, Metrics and Config) , thoughts?

Reply:
sounds good to me

Reply:
This sounds good @lenny-intel @cloudxxx8 will update PR this afternoon